### PR TITLE
Add io.github.lainsce.Notejot

### PR DESCRIPTION
**Pull request by lainsce**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by lainsce** [comment]
This one will EOL https://github.com/flathub/com.github.lainsce.notejot

**Comment by barthalion** [comment]
bot, build io.github.lainsce.Notejot

File: io.github.lainsce.Notejot.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+    "app-id" : "io.github.lainsce.Notejot",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.38",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "io.github.lainsce.Notejot",
+    "rename-icon" : "io.github.lainsce.Notejot",
```
Comment (bilelmoussaoui) [inline io.github.lainsce.Notejot.json:7]: Why this one is needed?

Comment (lainsce) [reply io.github.lainsce.Notejot.json:7]: Oh oops. Going to remove that. Was using an older manifest and renaming stuff around, missed that.

**Comment by barthalion** [comment]
/merge

Please follow the steps descriubed at https://github.com/flathub/flathub/wiki/App-Maintenance#end-of-life to mark the old ID as EOL.

**Pull request closed without merging**